### PR TITLE
Fix secrets_handling examples

### PR DESCRIPTION
**Pull request by jmikola**

### CODE REVIEW

File: .evergreen/secrets_handling/README.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ If using a Linux host on Evergreen, the shorthand version of the script can be u
 
 ```yaml
 - command: subprocess.exec
-    params:
+  params:
```
Comment (jmikola) [inline .evergreen/secrets_handling/README.md:39]: This indentation fix was missed in The https://github.com/mongodb-labs/drivers-evergreen-tools/commit/e17c99a96c0dc8620607f5929052267211d35946.

File: .evergreen/secrets_handling/README.md [hunk 2]
```diff
@@ -36,11 +36,11 @@ If using a Linux host on Evergreen, the shorthand version of the script can be u
...
     working_dir: src
     binary: bash
     args: |
-        ${DRIVERS_TOOLS}/.evergreen/atlas/setup-secrets.sh
+      ${DRIVERS_TOOLS}/.evergreen/atlas/setup-secrets.sh
```
Comment (jmikola) [inline .evergreen/secrets_handling/README.md:43]: I'm not convinced that the `args: |` line is correct.

`evergreen validate` reports:

```
ERROR: tasks section in 'subprocess.exec' command: parsing parameters for command 'subprocess.exec' (step 1 of 2): decoding mapstructure params: 1 error(s) decoding:

* 'args': source data must be an array or slice, got string
```

The docs for [subprocess.exec](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands/#subprocessexec) states that `args` should be a list.

Comment (blink1073) [reply .evergreen/secrets_handling/README.md:43]: Yeah, it should look like:

```
args:
          - ${DRIVERS_TOOLS}/...
```

I missed that when converting the examples from `shell.exec`.

Comment (jmikola) [reply .evergreen/secrets_handling/README.md]: Updated.

**Review by blink1073 — APPROVED** [review]
Thanks!   The OIDC failure is unrelated, fixed in #448

**Pull request merged**